tests/Unit/Extension/Propagator/XCloudTrace/UtilsTest.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Tests\Unit\Extension\Propagator\XCloudTrace;
+
+use OpenTelemetry\Extension\Propagator\XCloudTrace\Utils;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers \OpenTelemetry\Extension\Propagator\XCloudTrace\Utils
+ */
+class UtilsTest extends TestCase
+{
+
+    /**
+     * @dataProvider for_test_leftZeroPad
+     */
+    public function test_leftZeroPad(string $pad, int $howMuch, string $equalsTo) : void
+    {
+        $this->assertEquals(Utils::leftZeroPad($pad, $howMuch), $equalsTo, "Given leftZeroPad($pad, $howMuch) != $equalsTo");
+    }
+
+    public function for_test_leftZeroPad() : array {
```

Comment:
Project style for test names is:
```suggestion
    public function for_test_left_zero_pad() : array {
```

Reply:
Updated, repushed PR.